src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -66,27 +70,26 @@
   }
 }
 
-.player.casting .bottom-bar,
-.player.metadata-loaded.hover .bottom-bar,
-.player.state-paused .bottom-bar,
-.player.state-idle .bottom-bar,
-.player.ad-break .bottom-bar,
-.player.menu-active .bottom-bar {
-  opacity: 1;
-  visibility: visible;
-  height: auto;
-  padding: 6px 16px;
-}
-
-.player.overlay-active .gui-area .bottom-bar {
-  opacity: 0;
-  visibility: hidden;
-  height: 0;
+.player.casting,
+.player.metadata-loaded.hover,
+.player.state-paused,
+.player.state-idle,
+.player.ad-break,
+.player.menu-active {
+  &:not(.overlay-active){
+    .bottom-bar {
+      visibility: visible;
+      max-height: #{$bottom-bar-max-height}px;
+      &.line-break {
+        max-height: 92px;
+      }
+    }
+  }
 }
 
 .player.size-sm {
   .bottom-bar {
-    padding: 6px 8px;
+    padding: 0 $gui-small-gutter 4px;
```

Comment:
should be on left and right controls like in the top bar?

Reply:
You are right it is different. But thats the way it was and I didn't want to change the css completely